artifactory/services/groups.go

```diff
@@ -0,0 +1,89 @@
+package services
+
+import (
+	"encoding/json"
+	"fmt"
+	rthttpclient "github.com/jfrog/jfrog-client-go/artifactory/httpclient"
+	"github.com/jfrog/jfrog-client-go/auth"
+	"github.com/jfrog/jfrog-client-go/utils/errorutils"
+)
+
+type Group struct {
+	Name            string `json:"name,omitempty"`
+	Description     string `json:"description,omitempty"`
+	AutoJoin        bool   `json:"autoJoin,omitempty"`
+	AdminPrivileges bool   `json:"adminPrivileges,omitempty"`
+	Realm           string `json:"realm,omitempty"`
+	RealmAttributes string `json:"realmAttributes,omitempty"`
+}
+type GroupService struct {
```

Comment:
Let's make sure we have a single new line between the declaration of structs and functions.

Reply:
So, I know I made that change - I wonder if `gofmt` did this. Do you have a linter in place? I really don't want to continue to chase spacing issues

Reply:
We don't have a linter that currently fixes this. I definitely need something like this. We welcome any suggested solutions.